src/constants.ts

```diff
@@ -41,6 +44,10 @@ export class Constants {
     public static IoTHubAICreateDoneEvent = "AZ.IoTHub.Create.Done";
     public static IoTHubAIGetModuleTwinStartEvent = "AZ.Edge.ModuleTwin.Get.Start";
     public static IoTHubAIGetModuleTwinDoneEvent = "AZ.Edge.ModuleTwin.Get.Done";
+    public static IoTHubAIUpdateModuleTwinStartEvent = "AZ.Edge.ModuleTwin.Update.Start";
+    public static IoTHubAIUpdateModuleTwinDoneEvent = "AZ.Edge.ModuleTwin.Update.Done";
+    public static ModuleTwinJosnFileName = "azure-iot-module-twin.json";
+    public static ModuleTwinJosnFilePath = path.join(os.tmpdir(), Constants.ModuleTwinJosnFileName);

```

Comment:
the module twin file is per workspace right? If so, shall we keep the file in this structure? I am considering the case when two vs code windows are open

Reply:
Updated,